### ISSUE TITLE
Unhandled Rejection: CapacitorDataStorageSqlite plugin is not implemented on Android

### ISSUE THREAD

**Issue by dhwalin** [jepiqueau/capacitor-data-storage-sqlite#40]
Getting error:
```
Unhandled Rejection: CapacitorDataStorageSqlite plugin is not implemented on Android
```

Using Capacitor ^3.0.0

**Comment by jepiqueau** [comment]
@dhwalin Hi, I am out till next week so I did not upgraded to Capacitor 3.0.0 my plugins. It seems according to a note in Slack contributor that you need to update the gradle to release 7 in Android. Can you try and tell me if it works

**Comment by dhwalin** [comment]
I upgraded to 7 but other plugins also stopped working. I will give you update soon. Thank you.

**Comment by jepiqueau** [comment]
@dhwalin Everything is fixed now in 3.0.0 see also angular-data-storage-sqlite-app-starter which has also be upgraded to release 7.0 of Android gradle. so i close the issue
